pandas/tests/io/test_pytables.py

```diff
@@ -5167,3 +5168,71 @@ def test_dst_transitions(self):
                 store.append('df', df)
                 result = store.select('df')
                 assert_frame_equal(result, df)
+
+
+@pytest.fixture(scope='module')
+def pytables_hdf5_file():
+    """Use PyTables to create a simple HDF5 file."""
+
+    table_schema = {
+        'c0': tables.Time64Col(pos=0),
+        'c1': tables.StringCol(5, pos=1),
+        'c2': tables.Int64Col(pos=2),
+    }
+
+    t0 = time.time()
+
+    testsamples = [
+        {'c0': t0, 'c1': 'aaaaa', 'c2': 1},
+        {'c0': t0 + 1, 'c1': 'bbbbb', 'c2': 2},
+        {'c0': t0 + 2, 'c1': 'ccccc', 'c2': 10**5},
+        {'c0': t0 + 3, 'c1': 'ddddd', 'c2': 4294967295},
+    ]
+
```

Comment:
use ensure_clean_path here

Reply:
Done, commit upcoming.

Food for thought, I don't expect answers for this pull request here:

- If a test fails, don't we want to retain the corresponding HDF5 file in the file system to enable later inspection? If the answer is "yes" then we should not use a context manager for file deletion, but do it in the fixture teardown only upon test success.

- What do we think about using a pytest-provided fixture for temporary file management?

```python
@pytest.fixture()
def pytables_hdf5_file(tmp_path):
    [...]
```

In that case the file path would contain the test name, e.g. `tmp_path = PosixPath('PYTEST_TMPDIR/test_create_file0')` as documented here: https://docs.pytest.org/en/latest/tmpdir.html

Reply:
ensure_clean_path already does this, not really necessary to do anything else

Reply:
> ensure_clean_path already does this

The way I read the code it will _always_ remove the file, also upon test failure.

Anyway :).